src/services/dumpers/templates/agent-nodejs/javascript/models/model.hbs

```diff
@@ -2,7 +2,7 @@ const Mongoose = require('mongoose');
 
 const schema = new Mongoose.Schema({
 {{#each fields as |field|}}
-  {{wq field.name}}: {{#if field.ref}}{ type: {{field.type}}, ref: '{{field.ref}}' }{{else if (isObject field.type)}}{{>renderNested type=field.type level=1}}{{else}}{{field.type}}{{/if}},
+  {{wsc field.name}}: {{#if field.ref}}{ type: {{field.type}}, ref: '{{field.ref}}' }{{else if (isObject field.type)}}{{>renderNested type=field.type level=1}}{{else}}{{field.type}}{{/if}},
```

Comment:
This needs to be taken out from this PR, and done using a refactor PR. 

This removes the quote for the attribute names when it is not needed